draft-ietf-core-dns-over-coap.md

```diff
@@ -182,9 +182,15 @@ or automatic configuration, e.g., using a CoRE resource directory
 {{-ddr}}.
 Automatic configuration SHOULD only be done from a trusted source.
 
-Support for SVCB Resource Records {{-svcb}}, {{-svcb-dns}} or DNR Service Parameters {{-dnr}}
-are not specified in this document.
-{{-core-dnr}} explores solutions for CoAP for these mechanisms.
+A DoC server can also be discovered using SVCB Resource Records {{-svcb}}, {{-svcb-dns}} or DNR
+Service Parameters {{-dnr}}.
+{{-core-dnr}} \[TBD: change to draft-lenders-core-coap-dtls-svcb when published\] provides solutions
+to discover CoAP over (D)TLS servers using the "alpn" SvcParam. This document specifies "docpath" as
+a single-valued SvcParamKey whose value (in both presentation and wire format) MUST be a CoAP
+resource path. The DoC client MUST send any DoC request to the CoAP resource identified with the
+scheme associated with protocol identified with the "alpn" SvcParam or a comparable SvcParam, the
+authentication name as the host, the port from the "port" SvcParam (if present), and the path from
+the "docpath" SvcParam.
```

Comment:
How about:

```suggestion
resource path. The DoC client MUST send any DoC request to the CoAP resource by using the "docpath"
SvcParam to construct a resource identifier as described in \[TBD: draft-lenders-core-coap-dtls-svcb\].
```

? This way, the weird “the CoAP resource identified by using the "docpath" SvcParam […]” wording is fixed and we don't ossify ourselves to just URIs but also allow e.g. for CRIs ;-).

Reply:
WFM, but docpath itself being a URI reference already pretty much closes the door, whereas on the highlevel URIs and CRIs are equivalent anyway.